mindsdb/integrations/handlers/paypal_handler/README.md

```diff
@@ -85,6 +91,11 @@ Query Invoices_table:
 SELECT * FROM paypal_datasource.invoices
 ~~~~
 
+Query Subscriptions_table:
+~~~~sql
+SELECT * FROM paypal_datasource.subscription
```

Comment:
For the sake of consistency, let's update the table name to **subscriptions** (instead of subscription), throughout the code.